app/code/core/Mage/Core/etc/system.xml

```diff
@@ -1113,6 +1113,25 @@
                     </fields>
                 </design>
 
+                <global_search>
+                    <label>Global Search</label>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>0</show_in_website>
+                    <show_in_store>0</show_in_store>
+                    <sort_order>6</sort_order>
+                    <fields>
+                        <enable translate="label comment">
```

Comment:
Little suggestion - there is no comment :-)

Reply:
great eye! fixed